server/migrations/20211207195840-deprecate-offchain.js

```diff
@@ -0,0 +1,403 @@
+'use strict';
+
+const offChainCommunities = [
```

Comment:
We may want to have a way of dealing with any offchain communities that aren't in this list, as they could be spun up at any time between now and this code getting deployed.